readme.md

```diff
@@ -104,7 +111,7 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Your Answer:**
 
----
+* Its the time a password encryption will take to generate.
```

Comment:
Yes, also the amount of time you want to invest in making encryption more secure.